packages/core/test/stylable-transformer/at-property.spec.ts

```diff
@@ -98,4 +99,150 @@ describe('@property support', () => {
             },
         ]);
     });
+
+    describe('validation', () => {
+        it('should remove at property when used without a body', () => {
+            const config = {
+                entry: `/entry.st.css`,
+                files: {
+                    '/entry.st.css': {
+                        namespace: 'entry',
+                        content: `
+                        @property --x {
+                            syntax: '<color>';
+                            inherits: false;
+                            initial-value: #c0ffee;
+                        }
+                        
+                        @property --y;
+                        `,
+                    },
+                },
+            };
+
+            const result = expectWarningsFromTransform(config, []);
+
+            const declarations = result.meta.outputAst!.nodes;
```

Comment:
declaration is confusing. while a declaration might found at the top level, these are definitly at-rules or at least nodes